sklearn/tree/export.py

```diff
@@ -149,7 +149,12 @@ def plot_tree(decision_tree, max_depth=None, feature_names=None,
 
     fontsize : int, optional (default=None)
         Size of text font. If None, determined automatically to fit figure.
+     optional (default=None)
 
+    fontname : str,optional (default='helvetica')
```

Comment:
Does setting the default to `'sans'` address the issue?

Reply:
Set fontname= "sans" normal displays
But I did not change the default value.
 Only optional parameters were added.